Monika After Story/game/script-compliments.rpy

```diff
@@ -377,15 +377,18 @@ label mas_compliment_thanks:
     m 4ekbfa "I guess we're both lucky that we have each other, [player]~"
     menu:
         "You mean everything to me, [m_name]":
-            $ mas_gainAffection(10,bypass=True)
+            if not mas_isMoniLove():
```

Comment:
Issue with this:
If a player saved the compliment until `Love` then they're not going to get this affection boost.

Should just need a similar format to the other compliments so every user gets their affection gain properly.

Reply:
Will do my review after this is changed